packages/plugin-analytic-fb-pixel/src/types.ts

```diff
@@ -12,7 +12,6 @@ export interface PluginAnalyticFBPixelSettings {
 }
 
 declare module '@ima/core' {
-  // eslint-disable-next-line @typescript-eslint/no-empty-interface
```

Comment:
Seems like `no-empty-interface` has been replaced with `no-empty-object-type` which we now disable in global eslint config. We could keep the original behaviour by adjusting these comments to disable the new eslint rule instead of the removed `no-empty-interface`.

There are several more cases of this below.

Reply:
All next-line rules are back with `eslint-disable-next-line @typescript-eslint/no-empty-object-type` rule. `no-empty-object-type` rule in esling config is removed.